core/src/main/java/io/grpc/Server.java

```diff
@@ -38,4 +41,57 @@
  * application code or interceptors.
  */
 @ThreadSafe
-public abstract class Server {}
+public abstract class Server {
```

Comment:
We add "managed" to channel so that we could have "unmanaged" channels. I think the server will always be "managed" (means have life-cycle methods). If that's the case, I would vote for just `Server`.